src/wallet/silentpayments.cpp

```diff
@@ -0,0 +1,210 @@
+#include <wallet/silentpayments.h>
+#include <addresstype.h>
+#include <arith_uint256.h>
+#include <coins.h>
+#include <crypto/common.h>
+#include <crypto/hmac_sha512.h>
+#include <key_io.h>
+#include <undo.h>
+#include <logging.h>
+#include <pubkey.h>
+#include <policy/policy.h>
+#include <script/solver.h>
+#include <util/check.h>
+#include <wallet/wallet.h>
```

Comment:
In 644ff77be819a2dea15be0daad10fca8524a18f4 "Implement BIP352: Silent Payments"

`wallet/wallet.h` does not need to be included.

Reply:
Done